### PR TITLE
Fixes a getting started issue

### PR DESCRIPTION
**Pull request by tullo**
The README should probably also mention what route to take (/test-data) so noobs like me don't look like a big ? when nothing happens under /

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ The React render happens on both the server and the client using shared code. Re
 
 ```
 npm install
+npm run build
```
Comment (nkbt) [inline README.md]: I don't think build is necessary when you start server. Build is done in runtime.

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "build": "npm run clean && npm run build:webpack",
     "build:dev": "npm run clean && npm run build:webpack:dev",
     "start": "cross-env NODE_PATH=source babel-node devServer.js",
-    "server": "cross-env NODE_PATH=source babel-node source/server/index.js",
+    "server": "cross-env NODE_PATH=source babel-node source/server/server.js",
```
Comment (nkbt) [inline package.json]: ouch, I guess I've just merged another PR with the same fix =)

File: README.md (hunk 1 above)
Comment (tullo) [inline README.md]: After I cloned the project and npm install & start the browser reported a 404 for the referenced static file.

File: package.json (hunk 2 above)
Comment (ericelliott) [inline package.json]: Not sure what version this is diffing against. This is already in master.

File: README.md (hunk 1 above)
Comment (ericelliott) [inline README.md]: A 404 for which file, exactly?

**Comment by tullo** [comment]
The static/index.js gives the 404
```
   <!doctype html>
    <html>
      <head>
        <title>Your Cool React App!</title>
      </head>
      <body>
        <div id='root'><div data-reactid=".2gjcdesgb28" data-react-checksum="-51373845"><h1 data-reactid=".2gjcdesgb28.0"></h1></div></div>
        <script>
          window.BOOTSTRAP_CLIENT_STATE = {"books":{"items":[{"id":1,"text":"Book 1","count":2},{"id":2,"text":"Book 2","count":3},{"id":3,"text":"Book 3","count":4}]},"routing":{}}
        </script>
        <script src="/static/index.js"></script>
      </body>
    </html>
  ```

**Comment by tullo** [comment]
Steps to reproduce:
1. cd /tmp
2. git clone git clone https://github.com/cloverfield-tools/universal-react-boilerplate.git
3. cd universal-react-boilerplate/
4. npm i
5. npm start
6. firefox http://0.0.0.0:3000 or firefox http://0.0.0.0:3000/test-data
7. view-source:http://0.0.0.0:3000/test-data
8. follow the link to the script /static/index.js
9. the request above returns a 404 as the requested file should come from the build dir that does not exist yet
```
tree -L 1
├── app-home.js
├── circle.yml
├── devServer.js
├── LICENSE
├── node_modules
├── package.json
├── README.md
├── source
├── static
├── webpack.config.dev.js
└── webpack.config.prod.js
```
But the server side references (server/app.js) the ./build directory to serve '/static/index.js'
```
const buildDir = '/build';
const staticDir = path.join(settings.APP_HOME, buildDir);
app.use('/static', express.static(staticDir));
```

**Pull request merged**